src/ui/public/vis/components/tooltip/__tests__/positioning.js

```diff
@@ -147,6 +147,31 @@ describe('Tooltip Positioning', function () {
         }
       });
     });
+
+    it('ident only west overflow when tooltip overflows both sides of inner container but outer contains tooltip', function () {
```

Comment:
The inner container, $chart, is not centered in the window. $chart is flush with the $window on the left side. The click event is centered in the tooltip. I will add better commenting and an Expect to clear this up.

I abbr  'ident' avoid the 140 character limit. I could not figure out how to easily define